app/src/main/java/com/alexandruvalentinconstantin/newsreader/feature/newslist/model/EventModel.java

```diff
@@ -0,0 +1,11 @@
+package com.alexandruvalentinconstantin.newsreader.feature.newslist.model;
+
+public class EventModel {
```

Comment:
this event wrapper seems useless at this point, let's remove and use the ArticleItemViewModel directly